### PR TITLE
fix pyyaml install

### PR DESCRIPTION
**Pull request by MarkDaoust**

### CODE REVIEW

File: site/en/guide/keras.ipynb [hunk 1]
```diff
@@ -118,9 +118,22 @@
         "`tf.keras` makes TensorFlow easier to use without sacrificing flexibility and\n",
         "performance.\n",
         "\n",
-        "To get started, import `tf.keras` as part of your TensorFlow program setup:"
+        "To get started, install  import `tf.keras` as part of your TensorFlow program setup:"
```
Comment (lamberta) [inline site/en/guide/keras.ipynb]: either install or import

File: site/en/guide/keras.ipynb [hunk 2]
```diff
@@ -1003,22 +1016,9 @@
       },
       "cell_type": "markdown",
       "source": [
-        "Serializing a model to YAML format requires that you install `pyyaml`:"
+        "Serializing a model to YAML format requires that you install `pyyaml` *before you import tensorflow*:"
```
Comment (lamberta) [inline site/en/guide/keras.ipynb]: either \`import tensorflow\` or "import TensorFlow"

Comment (MarkDaoust) [reply site/en/guide/keras.ipynb]: oups.

**Comment by MarkDaoust** [comment]
Yes. Thanks.

**Pull request merged**